### DISCUSSION TITLE
why no examples of envelop with graphql-yoga

### DISCUSSION THREAD

**Discussion by ethdev279** [dotansimha/envelop#1454] [Q&A] [answered]
As you guys mention graphql-yoga is your recommended graphql server,  I'm quite surprised to see that there are no examples specific to yoga in the examples folder or your documentation. we need examples of using envelop's generic auth, public and private schema based on authentication, and examples of other plugins too. 

One more thing, for beginners like me, it's not clear where to use this `getEnveloped` variable in order to make it work. I don't see even one place in the documentation you specified this. pls, improve your docs. Thanks!

P.S: more examples on yoga+envelop and yoga+shield

```
const getEnveloped = envelop({
  plugins: [
    useLazyLoadedSchema(getSchema)
    // ... other plugins ...
  ]
})
```

**Comment by n1ru4l** [comment, marked as answer]
Hey @ethdev279, GraphQL Yoga already has envelop built-in. All you need to do is pass your plugins to the plugins option in yoga.
[This is documented on our Yoga docs.](https://www.graphql-yoga.com/docs/features/envelop-plugins)

[You can find an example using envelop `useGenericAuth` and yoga here.](https://github.com/dotansimha/graphql-yoga/tree/master/examples/generic-auth)
